.github/workflows/driver_test.yml

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) Microsoft Corporation
+# SPDX-License-Identifier: MIT
+
+# For documentation on the github environment, see
+# https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners
+#
+# For documentation on the syntax of this file, see
+# https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions
+name: Kernel_Test
+
+# Run this work flow when the "MSBuild workflow completes"
+on:
+  workflow_run:
+    workflows: ["MSBuild"]
+    types:
+      - completed
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        configurations: [Debug, Release]
+    runs-on: windows-latest
```

Comment:
is there any way to specify server?

Reply:
According to the link in line 5, "The windows-latest label currently uses the Windows Server 2019 runner image."